### PR TITLE
Add filtering on cluster/provisioner tables

### PR DESCRIPTION
**Pull request by ekhomyakova**
Depends on https://github.com/Mirantis/kqueen/pull/329

### CODE REVIEW

File: kqueen_ui/api.py [hunk 1]
```diff
@@ -177,8 +177,10 @@ def update(self, uuid, payload):
 class ListManager(BaseManager):
     resource_url = ''
 
-    def list(self, namespace=None, all_namespaces=None, page=1, per_page=20):
+    def list(self, namespace=None, all_namespaces=None, page=1, per_page=20, filters=None):
```
Comment (naumvd95) [inline kqueen_ui/api.py]: 1 question guys, do we accept None namespaces??

File: kqueen_ui/blueprints/ui/views.py [hunk 2]
```diff
@@ -869,10 +874,16 @@ class ClusterPage(KQueenView):
...
+        name_filter = request.args.get('cluster_name', '')
+        provisioner_filter = request.args.get('cluster_provisioner', '')
         try:
             clusters = self.kqueen_request(
                 'cluster', 'list',
-                fnkwargs={'page': page, 'per_page': self.objects_per_page}
+                fnkwargs={
+                    'page': page,
+                    'per_page': self.objects_per_page,
+                    'filters': {'name': name_filter, 'provisioner': provisioner_filter}
```
Comment (naumvd95) [inline kqueen_ui/blueprints/ui/views.py]: maybe 'cluster_name' ?

**Comment by katyafervent** [comment]
Good job! Please, remove wip from the commit message

**Comment by katyafervent** [comment]
Also, I suggest to hide filters if there are no or one raw in the table

**Comment by katyafervent** [comment]
Will we add engine selector to be able to show only provisioners for a particular type?

**Comment by katyafervent** [comment]
screenshot from 2018-07-02 14-15-12
Please, fix the message

**Comment by ekhomyakova** [comment]
> I suggest to hide filters if there are no or one raw in the table
Too many stuff to change to block filtering completely. I'm not sure that it worth to do.

> Will we add engine selector to be able to show only provisioners for a particular type?
Maybe later in a separate request?

**Pull request merged**